src/App.jsx

```diff
@@ -13,11 +10,8 @@ function App() {
       <h1>
         Todo List
       </h1>
-      <ul>
-        {todoList.map(todo => (
-          <li key={todo.id}>{todo.title}</li>
-        ))}
-      </ul>
+      <TodoList/>
+      <AddTodoForm/>
       </div>  
```

Comment:
Since we already have a fragment `<></>` that is the parent element for this component, having a div as the only child here is unnecessary. As long as we have one parent element in the return, we can give that element as many children as it wants!

Under the hood, a fragment `<></>` just inserts a blank div, so we basically will have two divs here holding the important content if you inspect the page.